ext/storage_sources/s3_store/S3StoreLogSystem.cxx

```diff
@@ -0,0 +1,73 @@
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/AWSLogging.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+#include "S3StoreLogSystem.h"
+
+#include <cstdarg>
+
+using namespace Aws::Utils;
+using namespace Aws::Utils::Logging;
+
+S3StoreLogSystem::S3StoreLogSystem(WT_EXTENSION_API *wtApi, uint32_t awsVerbose)
+{
+    static const std::map<int32_t, Aws::Utils::Logging::LogLevel> verbosityMapping = {
+      {-3, Aws::Utils::Logging::LogLevel::Error}, {-2, Aws::Utils::Logging::LogLevel::Warn},
+      {-1, Aws::Utils::Logging::LogLevel::Info}, {0, Aws::Utils::Logging::LogLevel::Info},
+      {1, Aws::Utils::Logging::LogLevel::Debug}};
+    if (verbosityMapping.find(awsVerbose) != verbosityMapping.end()) {
+        logLevel = verbosityMapping.at(awsVerbose);
+    } else {
+        logLevel = Aws::Utils::Logging::LogLevel::Error;
+    }
+    wt_api = wtApi;
+}
+
+void
+S3StoreLogSystem::Log(
+  Aws::Utils::Logging::LogLevel logLevel, const char *tag, const char *format, ...)
+{
+    Aws::StringStream ss;
+    std::va_list args;
+    va_start(args, format);
+    va_list tmpArgs; // unfortunately you cannot consume a va_list twice
```

Comment:
np: We usually use C style comments, i.e `/*...*/`. @sulabhM, we have not discussed about this, what is the preferred way?
Nonetheless, we start comments with a capital letter and a full stop.

Reply:
Please see the wiki for [General Guidelines for Commenting Code](https://github.com/mongodb/mongo/wiki/Style-Guidelines), and the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Comments). The `//` style is generally preferred for C++ code, but the important thing is to be consistent.